src/DataGrid.tsx

```diff
@@ -441,29 +712,14 @@ function DataGrid<R, K extends keyof R, SR>({
       )}
       {rows.length === 0 && emptyRowsRenderer ? createElement(emptyRowsRenderer) : (
         <>
-          {viewportWidth > 0 && (
-            <InteractionMasks<R, SR>
-              rows={rows}
-              rowHeight={rowHeight}
-              columns={columns}
-              enableCellAutoFocus={enableCellAutoFocus}
-              enableCellCopyPaste={enableCellCopyPaste}
-              enableCellDragAndDrop={enableCellDragAndDrop}
-              cellNavigationMode={cellNavigationMode}
-              eventBus={eventBus}
-              gridRef={gridRef}
-              totalHeaderHeight={totalHeaderHeight}
-              scrollLeft={scrollLeft}
-              scrollTop={scrollTop}
-              scrollToCell={scrollToCell}
-              editorPortalTarget={editorPortalTarget}
-              onCheckCellIsEditable={onCheckCellIsEditable}
-              onRowsUpdate={handleRowsUpdate}
-              onSelectedCellChange={onSelectedCellChange}
-            />
-          )}
           <div style={{ height: Math.max(rows.length * rowHeight, clientHeight) }} />
-          {getViewportRows()}
+          <div
+            className={clsx({ 'rdg-viewport-dragging': isDragging })}
+            onKeyDown={handleKeyDown}
+          >
+            {viewportWidth > 0 && getViewportRows()}
+            {getEditorContainer()}
```

Comment:
Maybe we should move this outside the div, so we can simplify `handleKeyDown`?

Reply:
Moved `onKeyDown` to the cell component